docs/Change_Log.md

```diff
@@ -1,10 +1,23 @@
 # Change Log
 
+## Updated Thread Permissions
+
+#### September 9th, 2021 (most likely)
+
+Thread permissions are being updated and simplified:
+
+- "Use Public Threads" is becoming "Create Public Threads", which will allow users to create threads in a channel, even if they cannot send messages in that channel
+- "Use Private Threads" is becoming "Create Private Threads", which will allow users to create threads in a channel, even if they cannot send messages in that channel
+
+A new permission is also being added:
+
+- "Send Messages in Threads", which will allow users to send a message in a thread. The "Send Messages" permission will have no effect in threads, users **must** have "Send Messages in Threads" to send a message in a thread. This allows for setups where a user can participate in a thread, but cannot send a message in the parent channel (like a thread off an announcement post).
```

Comment:
If a user has permission to create public threads, but not send messages in threads, would they still be able to create a thread?

Reply:
Yup!
From the changelog:
> - "Use Public Threads" is becoming "Create Public Threads", which will allow users to create threads in a channel, even if they cannot send messages in that channel

Reply:
Ah! Reading is hard 😢 Thank you.

Reply:
So they'd only be able to create a thread, but not sent messages in the thread after they create it?

Reply:
You could configure it that way if you wanted. idk why you would. But you could :)
There's a separate permission for sending in threads, "send messages in threads", so I would suspect most places would grant _both_ of those

It's similar to how you can have manage messages, but not send messages. It's _weird_, but also much simpler this way.

Reply:
Gotcha. So the option to create a thread with a starter message would not be possible, as that would still require sending a message in the thread?

![image](https://user-images.githubusercontent.com/10660608/130146335-c492a43c-5443-46c9-92f5-27a855b6071a.png)

Reply:
Both will still work
It's true it lets you send a message, but I'd argue that is part of the "act of creating a thread", so still covered by the Create Threads bit

Reply:
Perfect, thank you for clarifying all this!